### PR TITLE
Added method to calculate SNR using a single slice

### LINKED ISSUE #36: Add method for computing SNR from a single scan
Implement method as described in Griffanti et al., Biomed Sign Proc and Control, 2012.

https://www.sciencedirect.com/science/article/pii/S1746809411000565

### PR DESCRIPTION
**Pull request by stephaniealley**
To Do: calculate SNR_single slice-wise

Fixes #36

### CODE REVIEW

File: process_data.py [hunk 1]
```diff
@@ -62,21 +63,37 @@ def get_parameters():
...
+    :return: float: SNR rounded at 2 decimals
+    """
+    print("Compute SNR_single...")
+    # Convert image to array
+    data = Image(file_data).data
+    # Convert mask to array
+    wm_mask = Image(file_mask).data
+    roi= np.where(wm_mask == 1)
+    roi_data = data[roi]
+    snr_single = np.mean(roi_data) / np.std(roi_data)
```
Comment (jcohenadad) [inline process_data.py]: SNR should be computed slice wise

**Review by jcohenadad — CHANGES_REQUESTED** [review]

File: process_data.py (hunk 1 above)
Comment (stephaniealley) [reply process_data.py]: @jcohenadad yes, that's right. Adjusting it now.

**Comment by stephaniealley** [comment]
@jcohenadad SNR_single is now computed slice-wise

File: process_data.py [hunk 2]
```diff
@@ -62,21 +63,42 @@ def get_parameters():
...
+    """
+    print("Compute SNR_single...")
+    # Convert image to array
+    data = Image(file_data).data
+    # Convert mask to array
+    wm_mask = Image(file_mask).data
+    # Use mask to select data in ROI_body
+    roi_data = np.where(wm_mask == 1, data, 0)
+    # Compute SNR slice-wise
+    snr_slices = np.zeros((roi_data.shape[2]))
```
Comment (jcohenadad) [inline process_data.py:96]: if 3rd dimension is not z it will fail

**Review by jcohenadad — CHANGES_REQUESTED** [review]

File: process_data.py (hunk 2 above)
Comment (stephaniealley) [reply process_data.py:96]: Right, I see.

Comment (jcohenadad) [reply process_data.py:96]: use `im.change_orientation('RPI')`

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**